etl/steps/data/garden/un/2024-07-25/refugee_data.meta.yml

```diff
@@ -0,0 +1,279 @@
+  description_stateless: |-
+    Individuals under UNHCR’s statelessness mandate are defined under the 1954 Convention Relating to the Status of Stateless People as those not considered as nationals by any State under the operation of its law. In other words, they do not possess the nationality of any State. UNHCR statistics refer to people who fall under the organization’s statelessness mandate as those who are stateless according to this international definition. Data from some countries may also include people with undetermined nationality. These are people who lack proof of possession of any nationality and at the same time have or are regarded as having important links to more than one State. UNHCR also works with populations at risk of statelessness, but persons at risk of statelessness are not reported on under the statistical category of individuals under UNHCR’s statelessness mandate.
+  description_asylum: |-
+    Persons who have filed an application for asylum in a country other than their own and whose claims have not yet been determined.
+    These include those filing primary applications or subsequent applications following an appeal. The date on which the application for asylum is filed marks their entry into the status of asylum seeker.
+    They remain in the status of asylum seeker until their application is considered and adjudicated.
+  description_refugee: |-
+    Refugees include individuals recognized under the 1951 Convention relating to the Status of Refugees, its 1967 Protocol, the 1969 Organization of African Unity (OAU) Convention Governing the Specific Aspects of Refugee Problems in Africa, the refugee definition contained in the 1984 Cartagena Declaration on Refugees as incorporated into national laws, those recognized in accordance with the UNHCR Statute, individuals granted complementary forms of protection, and those enjoying temporary protection. The refugee population also includes people in refugee-like situations.
+  description_host_community: |-
+    Host community refers to a community that hosts large populations of refugees or internally displaced persons, whether in camps, integrated into households, or independently. UNHCR reports on host communities in countries with substantive programmes focusing on sharing the burden of hosting large refugee populations, as set out in the Global Compact on Refugees.
+  description_other_concern: |-
+   Other groups or persons of concern refers to individuals who do not necessarily fall directly into any of the other groups (refugees, asylum-seekers, stateless persons, internally displaced persons) but to whom UNHCR has extended its protection and/or assistance services, based on humanitarian or other special grounds.
+
+  description_return: |-
```

Comment:
I see these are descriptions from the producer, but include DoDs. In general they should be mostly the same as it's given by them https://docs.owid.io/projects/etl/architecture/metadata/reference/#variabledescription_from_producer

Reply:
Hm, I have interpreted the "description from producer" field as a more specific definition on how the producer defines something/ counts something in their collection of the data. The DoD I see as more of a general "dictionary" definition. Am I misunderstanding that?

Especially since here the description of the producer is for "returned refugees/ internally displaced people" and the DoD is just about refugees and internally displaced people, I see value in having both. Or would you remove the DoDs?